panel/src/components/Collection/Item.vue

```diff
@@ -48,8 +48,8 @@
 		>
 			<!-- Buttons -->
 			<k-button
-				v-for="(button, buttonIndex) in buttons"
-				:key="'button-' + buttonIndex"
+				v-for="button in buttons"
+				:key="JSON.stringify(button)"
```

Comment:
`Todo:` We should solve this already in v5 or on `develop/v6` before the Vue 3 merge. Ideally also not with `JSON.stringify`